app/javascript/components/widgets/widgets/forest-change/tree-cover-gain/selectors.js

```diff
@@ -80,18 +77,25 @@ export const parseData = createSelector(
 
 export const getSentence = createSelector(
   [getSortedData, getSettings, getIndicator, getCurrentLocation, getSentences],
-  (data, settings, indicator, currentLocation, sentences) => {
+  (data, settings, indicator, currentLabel, sentences) => {
     if (!data || !data.length) return null;
-    const { initial, withIndicator } = sentences;
+    const {
+      initial,
+      withIndicator,
+      globalWithIndicator,
+      globalInitial
+    } = sentences;
     const locationData =
-      currentLocation && data.find(l => l.id === currentLocation.value);
-    const gain = locationData && locationData.gain;
+      currentLabel && data.find(l => l.id === currentLabel.value);
```

Comment:
currentlLabel is a string not an object. This should fail. Use `currentLocation` if you need the value